icons/pizza.svg

```diff
@@ -9,9 +9,9 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M15 11h.01" />
-  <path d="M11 15h.01" />
-  <path d="M16 16h.01" />
-  <path d="m2 16 20 6-6-20c-3.36.9-6.42 2.67-8.88 5.12A19.876 19.876 0 0 0 2 16Z" />
-  <path d="M17 6c-6.29 1.47-9.43 5.13-11 11" />
+  <path d="M15 11h.01"/>
+  <path d="M11 15h.01"/>
+  <path d="M16 16h.01"/>
+  <path d="m2 16 20 6-6-20A20 20 0 0 0 2 16"/>
+  <path d="M17.22 6A15 15 0 0 0 6 17.2"/>
```

Comment:
The crust line is currently a bit too small, but with this change it's actually larger than it should be:
![image](https://user-images.githubusercontent.com/17746067/232278044-8cf71c3d-dbef-4c2e-9c5d-34c4799a28b4.png)

When fixing it, I realized that it's not even of the correct radius:
![image](https://user-images.githubusercontent.com/17746067/232278138-20cc48d9-5bee-4877-823f-9c431fee33ea.png)

This should work OK:
```suggestion
  <path d="M15 11h.01" />
  <path d="M11 15h.01" />
  <path d="M16 16h.01" />
  <path d="m2 16 20 6-6-20A20 20 0 0 0 2 16" />
  <path d="M5.71 17.11a17.04 17.04 0 0 1 11.4-11.4" />
```